docker/distelli-manifest.yml

```diff
@@ -1,4 +1,9 @@
 pe-and-platform/puppetserver:
+  PreBuild:
+    - bundle install --path .bundle/gems
+    - bundle exec ci/pre_build
   Build:
-    - docker login -u "$DISTELLI_DOCKER_USERNAME" -p "$DISTELLI_DOCKER_PW" "$DISTELLI_DOCKER_ENDPOINT"
-    - /bin/bash -x ./ci/build
+    - bundle exec ci/build
+    - bundle exec ci/test
+  AfterBuildSuccess:
+    - bundle exec ci/publish
```

Comment:
Tested these changes with only a changed namespace in the distelli-manifest at https://pipelines.puppet.com/release-engineering/builds/469030. The failure is expected, the job isn't set up with credentials to push to dockerhub.